src/containers/CreateCourse/CreateCourse.test.jsx

```diff
@@ -48,36 +48,45 @@ const initialState = {
 
 const middlewares = [thunk];
 const mockStore = configureStore(middlewares);
-let mockOnSubmit;
 let store;
 
 const createWrapper = (state) => {
   store = mockStore(state);
-  mockOnSubmit = jest.fn();
```

Comment:
Why do you remove this mock?

Reply:
This was unnecessary, `mockOnSubmit` was used to mock the `createCourse` behavior but there were no assertions being made. `CreateCoursePage` component was already defining `createCourse` as an empty function in its default props, so there was no need to mock it.

But if you still suggest bringing this back there are no objections from my side as it will not affect the router upgrade. :)